gnovm/cmd/gno/transpile.go

```diff
@@ -169,17 +195,39 @@ func execTranspile(cfg *transpileCfg, args []string, io commands.IO) error {
 	return nil
 }
 
-func transpilePkg(pkgPath importPath, opts *transpileOptions) error {
-	if opts.isTranspiled(pkgPath) {
+// transpilePkg transpiles all non-test files at the given location.
+// Additionally, it checks the gno.mod in said location, and skips it if it is
+// a draft module
+func transpilePkg(dirPath string, opts *transpileOptions) error {
+	if opts.isTranspiled(dirPath) {
+		return nil
+	}
+	opts.markAsTranspiled(dirPath)
+
+	gmod, err := gnomod.ParseAt(dirPath)
+	if err != nil && !errors.Is(err, gnomod.ErrGnoModNotFound) {
+		return err
+	}
+	if err == nil && gmod.Draft {
```

Comment:
```suggestion
        // If there is a gno.mod file, check if it is a Draft
	if gmod != nil && gmod.Draft {
```

IMHO might be clearer when trying to understand what is going on here.